protostar/migrator/cheatcodes/migrator_invoke_cheatcode.py

```diff
@@ -42,8 +39,7 @@ def from_dict(
             return cls()
         return cls(
             wait_for_acceptance=config.get("wait_for_acceptance", False),
-            max_fee=config.get("max_fee", None),
-            auto_estimate_fee=config.get("auto_estimate_fee", False),
+            max_fee=config.get("max_fee", "auto"),
```

Comment:
If `max_fee` is undefined, Protostar should throw an error to the user.

Note: Currently, `config` parameter is optional. We need to make it required for migration cheatcodes to force the user passing `max_fee`. However, this change will break the cheatcode Protocol between migration and testing module (unless we make it generic). This is a task for a different PR though.

Reply:
[make max_fee mandatory and update tests](https://github.com/software-mansion/protostar/pull/979/commits/b2e0a73bd1102a4cb0555cb2e3e91535b785d958)